budgetkey_data_pipelines/pipelines/entities/government_companies/registrar/pipeline-spec.yaml

```diff
@@ -0,0 +1,22 @@
+government_companies:
+  schedule:
+    crontab: "*/15 * * * *"
```

Comment:
This might be a bit too often - we can probably scan this weekly and still be ok.